apps/emqx_schema_registry/src/emqx_schema_registry.erl

```diff
@@ -51,13 +55,13 @@
 start_link() ->
     gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
 
--spec get_serde(schema_name()) -> {ok, serde_map()} | {error, not_found}.
+-spec get_serde(schema_name()) -> {ok, serde()} | {error, not_found}.
```

Comment:
Nit: I still think it's better to expose a map response in APIs in general rather than records like `#serde{}`, so it's not necessary to know its definition and it's easier to evolve.

Reply:
I meant that maps are more convenient than records for APIs in general, for callers of this module.